keps/sig-release/3000-artifact-distribution/kep.yaml

```
@@ -0,0 +1,24 @@
+title: Artifact Distribution Policy
+kep-number: 3000
+authors:
+  - "@hh"
+  - "@BobyMCbobs"
+owning-sig: sig-release
+participating-sigs:
+  - sig-k8s-infra
+status: provisional
+creation-date: 2021-11-26
+reviewers:
+  - "@cpanato"
+  - "@puerco"
+  - "@spiffxp"
+  - "@thockin"
+approvers:
+  - "@ameukam"
+  - "@justaugustus"
+stage: alpha
+latest-milestone: "v1.24"
+milestone:
+  alpha: "v1.24"
+  beta: "v1.25"
+  stable: "v1.26"
```

We should update this to reflect the current progress we make in https://github.com/kubernetes-sigs/promo-tools/issues/533

```suggestion
milestone:
  alpha: "v1.24"
  beta: "v1.25"
  stable: "v1.26"
```

---

Updated in 0e5fafb10766746015900d914606e9a5ea657733